roles/openshift_project_request_template/README.md

```diff
@@ -0,0 +1,33 @@
+OpenShift Project Request Template
+==================================
+
+Configure template used when creating new projects. If enabled only the template is managed. It must still be enabled in the OpenShift master configuration. The base template is created using `oc adm create-bootstrap-project-template` and can be modified by setting `openshift_project_request_template_edits`.
```

Comment:
Is this intended to be used for all projects that are created, or just some?

Reply:
It's meant to configure the cluster-wide project template (configurable via `osm_project_request_template`).